opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/OpenTracingShim.java

```diff
@@ -18,16 +18,23 @@
   private OpenTracingShim() {}
 
   /**
-   * Creates a {@code io.opentracing.Tracer} shim out of {@code OpenTelemetry.getTracerProvider()}
-   * and {@code OpenTelemetry.getPropagators()}.
+   * Creates a {@code io.opentracing.Tracer} shim out of {@code
+   * GlobalOpenTelemetry.getTracerProvider()} and {@code GlobalOpenTelemetry.getPropagators()}.
    *
    * @return a {@code io.opentracing.Tracer}.
    */
   public static io.opentracing.Tracer createTracerShim() {
-    return new TracerShim(
-        new TelemetryInfo(
-            getTracer(GlobalOpenTelemetry.getTracerProvider()),
-            GlobalOpenTelemetry.getPropagators()));
+    return createTracerShim(getTracer(GlobalOpenTelemetry.getTracerProvider()));
```

Comment:
I don't see any way here to actually use custom propagators. As far as I can see, we only every use the standard stuff in `GlobalOpenTelemetry` or the `OpenTelemetry` instance passed in directly. Are we missing a creational method here that will allow passing in custom `OpenTracingPropagators` somewhere?

Reply:
added